repos.yaml

```diff
@@ -1,10 +1,10 @@
 # These are the repos to be downloaded with vcstool
 repositories:
-  isc_joy:
+  include/isc_joy:
    type: git
    url: https://github.com/iscumd/isc_joy
    version: master
-  robot_state_controller:
+  include/robot_state_controller:
```

Comment:
any specific reason you're doing this? typically the catkin workspace will have just the catkin packages, not an include folder with catkin packages

Reply:
I was doing this just so it was clear which files were a part of the Ohm repo, so people don't attempt to edit the external nodes like robot state machine locally and cause their local workspace to become out of date with origin, as the nodes added with vcstool won't be commited. 

If this is too far out of ROS conventions I can reverse it, It just seemed like a nice way to cleanup the repo.

Reply:
It's very atypically for ROS conventions; however, there is no specific rule on how it should be done. External dependencies are installed through apt, meaning they exist as binaries in your /opt/ros folder. The things that are being installed this way may also be technically considered external, but each of the repos is cloned and is a functional git repo. It's up to you if you want to structure it this way, but I would advise against it.

Reply:
Alright, the two remote repos are now top level again.